hack/machineset.sh

```diff
@@ -187,7 +187,6 @@ get_gcp_ms() {
   # Installer created MachineSets only append the `-a` portion, so we should do the same.
   local az_suffix=$(echo $az |awk -F "-" '{print $NF}')
   local projectID=$(gcloud config get-value project)
```

Comment:
If the intent is remove the dependency on using `glcoud`, you still need it to find the project ID. Can we hard code this? Or would that be leaking some info?